Cargo.toml

```diff
@@ -13,12 +13,12 @@ keywords = ["falco", "falcosecurity", "kubernetes", "security"]
 [dependencies]
 grpcio = { version = "0.4.7", features = ["openssl"] }
 failure = "0.1"
-futures = "^0.1.15"
-protobuf = "~2"
+futures = "^0.1.29"
+protobuf = "~2.17"
```

Comment:
```
error[E0658]: non-builtin inner attributes are unstable                                                               │
 --> /home/vagrant/.cargo/registry/src/github.com-1ecc6299db9ec823/protobuf-2.17.0/src/well_known_types/type_pb.rs:9:1│
  |                                                                                                                   │
9 | #![rustfmt::skip]                                                                                                 │
  | ^^^^^^^^^^^^^^^^^                                                                                                 │
  |                                                                                                                   │
  = note: for more information, see https://github.com/rust-lang/rust/issues/54726
```

It seems to me that protobuf 2.17 contains unstable features (that only works with an unstable `rustc`).

Reply:
Was this version bump mandatory?

If yes, why?

Thanks in advance :)

Reply:
I don't think it was mandatory, I did it because it seemed like a convenient time to update the version